raft/src/main/java/org/apache/kafka/raft/ReplicatedCounter.java

```diff
@@ -107,6 +107,8 @@ public synchronized void handleCommit(BatchReader<Integer> reader) {
             }
             log.debug("Counter incremented from {} to {}", initialCommitted, committed);
 
+            // A snapshot is being taken here too, not being able to -
+            // `import org.apache.kafka.metadata.utils.SnapshotReason`, figure out why?
```

Comment:
The `raft` module only relies on the `clients` module so it can't import a class from the `metadata` module.

Reply:
I see, so will we ignore this snapshot request and not log the reason for it?

Reply:
Here we already logged the reason: "the previoud snapshot includes {}", please remove the new comments and help to fix the typo "previoud".